components/automate-ui/src/app/pages/client-runs/client-runs.component.ts

```diff
@@ -245,29 +249,40 @@ export class ClientRunsComponent implements OnInit, OnDestroy {
     this.currentPage$ = this.store.select(createSelector(clientRunsState,
       (state) => state.nodeFilter.page));
 
-    this.selectedStatus$ = this.store.select(createSelector(clientRunsState,
-      (state) => state.nodeFilter.status));
+    this.selectedStatuses$ = this.searchBarFilters$.pipe(
+      map((chicklets: Chicklet[]) => chicklets.filter(chicket => chicket.type === 'status').
+        map(chicket => chicket.text)));
 
     this.totalNumberOfNodesWithStatusFilter$ = combineLatest(
-      this.selectedStatus$, this.nodeCounts$)
+      this.selectedStatuses$, this.nodeCounts$)
       .pipe(
-        map(([status, nodeCount]) => {
-          switch (status) {
-            case 'success':
-              return nodeCount.success;
-            case 'failure':
-              return nodeCount.failure;
-            case 'missing':
-              return nodeCount.missing;
-            default:
-              return nodeCount.total;
+        map(([statuses, nodeCount]) => {
+          if (statuses.length === 0 ) {
+            return nodeCount.total;
+          } else {
+            let count = 0;
+            if (statuses.findIndex(status => status === 'success') >= 0) {
+              count = count + nodeCount.success;
+            }
+            if (statuses.findIndex(status => status === 'failure') >= 0) {
+              count = count + nodeCount.success;
+            }
+            if (statuses.findIndex(status => status === 'missing') >= 0) {
+              count = count + nodeCount.success;
+            }
+
+            return count;
           }
         }));
 
-    this.isMissingStatusSelected$ = this.selectedStatus$.pipe(map(status => status === 'missing'));
-    this.isSuccessStatusSelected$ = this.selectedStatus$.pipe(map(status => status === 'success'));
-    this.isFailureStatusSelected$ = this.selectedStatus$.pipe(map(status => status === 'failure'));
-    this.isTotalStatusSelected$ = this.selectedStatus$.pipe(map(status => status === undefined));
+    this.isMissingStatusSelected$ = this.selectedStatuses$.pipe(map(statuses =>
+      statuses.findIndex(status => status === 'missing') >= 0));
+    this.isSuccessStatusSelected$ = this.selectedStatuses$.pipe(map(statuses =>
+      statuses.findIndex(status => status === 'success') >= 0));
+    this.isFailureStatusSelected$ = this.selectedStatuses$.pipe(map(statuses =>
+      statuses.findIndex(status => status === 'failure') >= 0));
+    this.isTotalStatusSelected$ = this.selectedStatuses$.pipe(map(statuses =>
+      statuses.length === 0));
```

Comment:
These above changes are to account that multiple statuses can be selected. Unlike before only one status could have been selected.